utils/db_utils.py

```diff
@@ -140,6 +142,10 @@ def query_trajectories(start_date, end_date):
     df.drop(columns= columns_to_drop, inplace=True) 
     modified_columns = perm_utils.get_trajectories_columns(df.columns)  
     df.columns = modified_columns 
+    for col in constants.EXCLUDED_TRAJECTORIES_COLS:
+        if col in df.columns:
+            df.drop(columns= [col], inplace=True) 
+    df['data.mode'] = df['data.mode'].apply(lambda x: ecwm.PredictedModeTypes(x).name if x in set(enum.value for enum in ecwm.PredictedModeTypes) else 'UNKNOWN')
```

Comment:
`mode` is a `MotionTypes` not a `PredictedModeTypes`
Please make sure to understand and double-check the OpenPATH data model
https://github.com/e-mission/e-mission-server/blob/bc78230d3196ca10249b6e95898f63fddf924f6f/emission/core/wrapper/recreatedlocation.py#L20